app/src/main/java/org/p2p/wallet/settings/ui/settings/presenter/SettingsItem.kt

```diff
@@ -0,0 +1,35 @@
+package org.p2p.wallet.settings.ui.settings.presenter
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+
+sealed interface SettingsItem {
+    data class SettingsGroupTitleItem(
+        @StringRes val groupTitleRes: Int
+    ) : SettingsItem
+
+    data class TextSettingItem(
```

Comment:
Setting doesn't mean the single element of settings
Let's do it `TextSettingsItem` and user it on other types